README.md

```diff
@@ -27,15 +27,21 @@ To create a basic chatbot using LlmComposer, you need to define a module that us
 Application.put_env(:llm_caller, :openai_key, "<your api key>")
 
 defmodule MyChat do
-  use LlmComposer.Caller,
+
+  @settings %LlmComposer.Settings{
     model: LlmComposer.Models.OpenAI,
     model_opts: [model: "gpt-4o-mini"],
-    system_prompt: "you are a helpful assistant."
+    system_prompt: "You are a helpful assistant."
+  }
+
+  def simple_chat(msg) do
+    LlmComposer.simple_chat(@settings, msg)
+  end
 end
 
 {:ok, res} = MyChat.simple_chat("hi")
 
-IO.inspect res.main_response
+IO.inspect(res.main_response)
```

Comment:
It would be handy to include an example of response.

Reply:
It is some lines below